docs/architecture/decisions/adr-018.md

```diff
@@ -0,0 +1,25 @@
+# ADR 18: Use external layout information
+
+## Context
+
+Many accessibility checks depend on the precise layout of the page. Most notably to detect the background color of text positioned on top of an image, but also to detect whether content is moved offscreen, or inspect the size of some piece of content. The lack of layout information has been a recurring issue as exemplified in [this list of problems](https://github.com/Siteimprove/alfa/issues/1256). Heuristics can only go so far…
+
+There are two main approaches to add layout information to Alfa:
```

Comment:
**suggestion (optional):** not sure about the grammar, but this sounds more natural to me
```suggestion
There are two main approaches to adding layout information to Alfa:
```